docs/docs/quickstart/installation/installation.md

```diff
@@ -11,21 +11,24 @@ include path.
 
 1. Copy `graaflip` to your project.
 2. Before compiling, add the directory to your include path.
-    ```bash
-    # For C compiler
-    export C_INCLUDE_PATH="/full/path/to/include/graaflib/:$C_INCLUDE_PATH"
-    # For Cpp compiler
-    export CPLUS_INCLUDE_PATH="/full/path/to/include/graaflib/:$CPLUS_INCLUDE_PATH"
-    ```
+
+   ```bash
+   # For C compiler
+   export C_INCLUDE_PATH="/full/path/to/include/:$C_INCLUDE_PATH"
+   # For Cpp compiler
+   export CPLUS_INCLUDE_PATH="/full/path/to/include/:$CPLUS_INCLUDE_PATH"
+   ```
 
    Or in CMake:
-    ```CMake
-    include_directories("graaf/src/graaflib")
-    ```
+
+   ```CMake
+   include_directories("graaf/include")
+   ```
+
 3. Include the graaf header in your sources.
-    ```c++
-    #include <graaflib/directed_graph.h>
-    ```
+   ```c++
+   #include <graaflib/directed_graph.h>
+   ```
```

Comment:
Previously this was outdated 😅 We have now moved the definition of all graph classes into `graaflib/graph.h`.
Would be nice if we could fix this as well:
```suggestion
   ```c++
   #include <graaflib/graph.h>
   ```
```